tools/test.sh

```diff
@@ -22,6 +22,9 @@ export INTEGREAT_CMS_DEEPL_AUTH_KEY="dummy"
 # Disable linkcheck listeners during testing
 export INTEGREAT_CMS_LINKCHECK_DISABLE_LISTENERS=1
 
+# Disable background tasks during testing
+export INTEGREAT_CMS_BACKGROUND_TASKS_ENABLED=0
```

Comment:
This also needs to be added to the [circleci_settings.py](https://github.com/digitalfabrik/integreat-cms/blob/2c560214ffb3a8af52435dec32192d7dca730a21/integreat_cms/core/circleci_settings.py) (the `test.sh` is only used for local testing)